src/plugins/auto/infer_request.cpp

```diff
@@ -91,13 +91,6 @@ void MultiDeviceInferRequest::SetBlob(const std::string& name, const InferenceEn
         IInferRequestInternal::SetBlob(name, blob);
 }
 
-void MultiDeviceInferRequest::SetBlob(const std::string& name, const Blob::Ptr& blob, const PreProcessInfo& info) {
```

Comment:
Can we remove SetBlob according to https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/ie/cpp/ie_infer_request.hpp Line 117

Reply:
https://docs.openvino.ai/latest/openvino_docs_OV_UG_Preprocessing_Overview.html
need to under stand preprocess steps

Reply:
Updated